test/unit/SourceLocator/Type/Composer/Factory/MakeLocatorForComposerInstalledJsonTest.php

```diff
@@ -114,6 +115,24 @@ public function expectedLocators() : array
                 $projectA,
                 $projectALocator,
             ],
+            [
+                $projectComposerV2,
+                new AggregateSourceLocator([
+                    new PsrAutoloaderLocator(
+                        Psr4Mapping::fromArrayMappings([
+                            'A\\B\\'        => [
+                                $projectComposerV2 . '/vendor/a/b/src/ab_PSR-4_Sources',
+                            ],
+                        ]),
+                        $astLocator
+                    ),
+                    new PsrAutoloaderLocator(
+                        Psr0Mapping::fromArrayMappings([]),
+                        $astLocator
+                    ),
+                    new DirectoriesSourceLocator([], $astLocator),
```

Comment:
It's not clear to me why either of the above two loaders exist, particularly the empty Psr0 one. But if anything this seems like a bug unrelated to my change, so I just added them here to the expected result.

Reply:
This comes from the `"files"` and `"classmap"` configurations, but since you don't have any, it's empty by design.

Reply:
Would have assumed it would not generate those locators at all then, but okay ;-) Not like I really read more of this code anyway :-D

Reply:
An empty locator is no-op anyway: be it there or not makes no big difference :+1: